### PR TITLE
[T-117] Product models defined for API communication between core and RS

### PR DESCRIPTION
**Pull request by matejscerba**

### CODE REVIEW

File: src/recommender_system/app/models/product.py
Comment (michalpulpan) [inline src/recommender_system/app/models/product.py:1]: Maybe here should be definition of `locale` or `language`?

Comment (matejscerba) [reply src/recommender_system/app/models/product.py:1]: I will probably use just one language at the moment, but I should add it there to be used in the future.

File: src/recommender_system/app/models/product.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+class Product(BaseModel):
+    """
+    This model represents product as an object that is sent from core to RS
+    component via API.
+    """
+
+    id: str
+    published: bool
+    category_id: int
+    product_translations: List[ProductTranslation]
```
Comment (michalpulpan) [inline src/recommender_system/app/models/product.py:19]: beautiful

**Review by michalpulpan — APPROVED** [review]
💘

**Review by lakatop — APPROVED** [review]

**Pull request merged**